charts/jaeger/templates/es-index-cleaner-cronjob.yaml

```diff
@@ -44,10 +44,7 @@ spec:
             - {{ .Values.esIndexCleaner.numberOfDays | quote }}
             - {{ include "elasticsearch.client.url" . }}
             env:
-              {{- range $key, $value := .Values.esIndexCleaner.env }}
-              - name: {{ $key | quote }}
-                value: {{ $value | quote }}
-              {{- end }}
```

Comment:
This was not advertised in the values file, also it is non-critical so I am fine with removing it in this PR.